engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/ConfluentKafkaAvroSerializer.scala

```diff
@@ -14,14 +14,10 @@ import scala.collection.JavaConverters._
 
 /**
   * This is Kafka Avro Serializer class. All events will be serialized to provided schema.
-  *
-  * @Important: there can be some delay between saved process schema and deploy schema, because
-  *            fetching schema for serializer has place at deploy moment. It can be happen when process has
-  *            set latest version and deploy was run after new schema was added.
   */
-class ConfluentKafkaAvroSerializer[T](kafkaConfig: KafkaConfig, confluentSchemaRegistryClient: ConfluentSchemaRegistryClient, schemaEvolutionHandler: AvroSchemaEvolution,
-                                      schemaOpt: Option[AvroSchema], var isKey: Boolean)
-  extends AbstractConfluentKafkaAvroSerializer(schemaEvolutionHandler) with Serializer[T] {
+class ConfluentKafkaAvroSerializer(kafkaConfig: KafkaConfig, confluentSchemaRegistryClient: ConfluentSchemaRegistryClient, schemaEvolutionHandler: AvroSchemaEvolution,
+                                   schemaOpt: Option[AvroSchema], var isKey: Boolean)
```

Comment:
Let's replace everywhere  `schemaOpt: Option[AvroSchema]` by `avroSchemaOpt: Option[AvroSchema]`. WDYT?

Reply:
> Some unused elements:
> 
> * `BaseKafkaAvroSourceFactory.createSource.version`
> * `BaseKafkaAvroSourceFactory.processObjectDependencies`

Good point, will fix it